Exec/science/subchandra/problem_initialize_state_data.H

```diff
@@ -52,6 +52,15 @@ void problem_initialize_state_data (int i, int j, int k,
         burn_state.xn[n] = state(i,j,k,UFS+n);
     }
 
+#ifdef NSE_TABLE
```

Comment:
I think it's better if we use the `AUX_THERMO` define here, which is set when `NSE_TABLE` is set.  That makes it clear that this is just about setting the aux quantities.

Reply:
done